source/Ship.h

```diff
@@ -618,6 +618,11 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Acceleration can be created by engines, firing weapons, or weapon impacts.
 	Point acceleration;
 
+	// Jump Damage: damage dealt after jumping.
```

Comment:
It would seem like it would be better to have the damage be done while jumping no?

Reply:
yea, did that